clar2wasm/tests/wasm-generation/values.rs

```diff
@@ -61,20 +61,39 @@ mod clarity_v2_v3 {
     use clarity::vm::Value;
 
     use super::*;
-    use crate::runtime_config;
 
     proptest! {
-        #![proptest_config(runtime_config())]
-
         #[test]
-        fn value_serialized_and_deserialized(val in PropValue::any().prop_filter("Filter condition description", |val| {
+        fn value_serialized_and_deserialized(val in PropValue::any()) {
             let mut env = TestEnvironment::default();
-            env.evaluate(&format!("(to-consensus-buff? {val})")).is_ok()
-        })) {
-            crosscheck(
-                &format!("(from-consensus-buff? {} (unwrap-panic (to-consensus-buff? {})))", val.type_string() ,val),
+
+            let to_consensus_snippet = format!("(to-consensus-buff? {val})");
+            let pre_check = env.evaluate(&to_consensus_snippet);
+
+            // Discard test cases only when `to-consensus-buff?` evaluation
+            // could not determine the type of the input parameter.
+            // For instance, `(to-consensus-buff? (list))` where a `NoType` should not be evaluated.
+            let err_msg = "could not determine the input type for the serialization function";
+            prop_assume!(match pre_check {
+                Ok(_) => true,
+                Err(ref e) if e.to_string().contains(err_msg) => false,
+                _ => true
+            });
+
+            let snippet = format!(
+                "(from-consensus-buff? {} (unwrap-panic (to-consensus-buff? {})))",
```

Comment:
It feels to me that there could be a problem here. If the `to-consensus-buff?` operation returns `None`, the function panics and the test fails.

Reply:
Yep, remove that test.